src/Microsoft.Azure.WebJobs.Extensions.Kafka/Trigger/SingleItemFunctionExecutor.cs

```diff
@@ -22,6 +22,7 @@ public class SingleItemFunctionExecutor<TKey, TValue> : FunctionExecutorBase<TKe
         public SingleItemFunctionExecutor(ITriggeredFunctionExecutor executor, IConsumer<TKey, TValue> consumer, int channelCapacity, int channelFullRetryIntervalInMs, ICommitStrategy<TKey, TValue> commitStrategy, ILogger logger)
             : base(executor, consumer, channelCapacity, channelFullRetryIntervalInMs, commitStrategy, logger)
         {
+            logger.LogInformation($"FunctionExecutor Loaded: {nameof(MultipleItemFunctionExecutor<TKey, TValue>)}");
```

Comment:
I think we want to use  `nameof(SingleItemFunctionExecutor<TKey, TValue>` here

Reply:
Done